frontend/imports/CodemirrorPlutoSetup.js

```diff
@@ -42,9 +42,129 @@ import {
     indentUnit,
     combineConfig,
     NodeProp,
+    parseMixed,
+    markdown,
+    html,
+    htmlLanguage,
+    markdownLanguage,
+    javascript,
+    javascriptLanguage,
+    sql,
+    PostgreSQL,
+    python,
+    pythonLanguage,
     autocomplete,
 } from "https://cdn.jsdelivr.net/gh/JuliaPluto/codemirror-pluto-setup@0.19.4/dist/index.es.min.js"
 
+const htmlParser = htmlLanguage.parser
```

Comment:
Can you put all additions to this file in a new file? Right now we don't get typings for the code inside this file and that makes it hard to work with. (Did you write this without typings??)